lib/compress/zstd_cwksp.h

```diff
@@ -310,7 +331,7 @@ ZSTD_cwksp_internal_advance_phase(ZSTD_cwksp* ws, ZSTD_cwksp_alloc_phase_e phase
                 ws->tableEnd = objectEnd;  /* table area starts being empty */
                 if (ws->tableValidEnd < ws->tableEnd) {
                     ws->tableValidEnd = ws->tableEnd;
-        }   }   }
+                }   }   }
```

Comment:
nit: doesn't seem to be the proper bracket alignment

Reply:
Fixed